source/query-bar.txt

```diff
@@ -53,24 +53,20 @@ operator.
 
 .. tip:: 
 
-   In the :doc:`Schema </schema>` tab, you can also use the Query
-   Builder to enter a query into the query bar. For details, see
-   :ref:`build-query`.
+   In the :doc:`Schema </schema>` tab, you can also use the :ref:`build-query`
+   to enter a query into the query bar.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
-</reference/mongodb-extended-json/>` types. For instance, to specify an
-``ObjectID``, use the format ``{ "$oid": "<string>" }``
-as in the following filter example:
+</reference/mongodb-extended-json/>` types. To specify an
+`ObjectID <https://docs.mongodb.com/manual/reference/method/ObjectId/>`_,
```

Comment:
For future reference -- if you look at line 61-62, that's another way to link to the manual.  Don't bother updating this commit.   More for you to know in future.